libraries/AP_HAL_ChibiOS/hwdef/revo-mini-sd/hwdef.dat

```diff
@@ -0,0 +1,162 @@
+# hw definition file for processing by chibios_pins.py
+# for Revo-Mini hardware
+
+# MCU class and specific type
```

Comment:
Same here - please instead include the hwdef on which this is based, undef the pins you want to change and then put in the definitions you want.

See KakuteF4Mini for an example